### ISSUE TITLE
Using torch autocast

### ISSUE THREAD

**Issue by AAnoosheh** [CompVis/taming-transformers#111]
Hi,

Can someone confirm if the model is known to work in mixed-precision by simply using torch's autocast function?
(or if there's a more proper way intended)

Thanks

**Comment by kaihe** [comment]
got nan loss trying to use pytorch-lightning mix precision API

```
trainer_kwargs["precision"] = 16
trainer_kwargs["amp_backend"] = "native"
```

**Comment by joanrod** [comment]
@kaihe Did you figure out how to fix the NaN issue? Having the same NaN loss problem.

**Comment by kaihe** [comment]
@joanrod I give it up mix precision training

**Comment by rishavpramanik** [comment]
> @kaihe Did you figure out how to fix the NaN issue? Having the same NaN loss problem.

I also don't find the mixed precision to be good
